components/search/bills/BillSearch.tsx

```diff
@@ -30,6 +31,31 @@ const searchClient = new TypesenseInstantSearchAdapter({
   }
 }).searchClient
 
+const extractLastSegmentOfRefinements = (items: any[]) => {
```

Comment:
nit: Could we put a more specific type on `items` than `any`?

Reply:
And I feel like we could put a more specific name on this function since it doesn't apply to all refinements? It's specifically for cleaning up the labels for HierarchicalMenu refinements - so maybe something more like `displayLowestTierForHierarchical` / `extractLowestTierForHierarchical`?